### PR TITLE
Fix spelling issues in Operator Whitepaper

### PR DESCRIPTION
**Pull request by tohjustin**
- Standardize all words to use U.S. English spelling as recommended by the [Kubernetes Documentation Style Guide](https://kubernetes.io/docs/contribute/style/style-guide/)
- Fix various spelling errors & missing punctuations
- Fix various formatting issues (i.e. trailing spaces)

### CODE REVIEW

**Review by Jenniferstrej — APPROVED** [review]

File: operator-wg/whitepaper/Operator-WhitePaper_v1-0.md [hunk 1]
```diff
@@ -137,8 +137,8 @@ By putting these common operational tasks into code, it can be ensured they will
 
 #### Domain Knowledge
 Similar to operational automation, it can be written into an operator to encode specialized domain knowledge about particular software or processes. A common example of this is application upgrades. While a simple stateless application might need nothing more than a Deployment’s rolling upgrade; databases and other stateful applications often require very specific steps in sequence to safely perform upgrades. The operator can handle this autonomously as it knows your current and requested versions and can run specialized upgrade code when needed. More generally, this can apply to anything a pre-cloud-native environment would use manual checklists for (effectively using the operator as an executable runbook).
-Another common way to take advantage of automated domain knowledge is error remediation. For example, the Kubernetes built-in remediation behaviours mostly start and end with “restart container until it works” which is a powerful solution but often not the best or fastest solution.
```
Comment (Jenniferstrej) [inline operator-wg/whitepaper/Operator-WhitePaper_v1-0.md:140]: I see that a few changes here are to change the English writing from British to American. I need to double-check if it's consistent with the rest of the whitepaper.

**Review by Jenniferstrej — APPROVED** [review]

**Review by thschue — APPROVED** [review]
lgtm

**Comment by tohjustin** [comment]
Hi @OmerKahani, by any chance you have time this week to review this PR? 🙇‍♂️

**Comment by tohjustin** [comment]
Thanks everyone for reviewing! 🙏

**Pull request merged**